dev/com.ibm.ws.transaction/resources/OSGI-INF/l10n/metatype.properties

```diff
@@ -93,3 +93,7 @@ lease.length.desc=The length of time before a recovery log lease expires
 
 lease.renewal.threshold=Lease renewal threshold
 lease.renewal.threshold.desc=The percentage of the duration of the recovery log lease that passes before the lease is renewed
+
+enable.log.retries=Enable retries on database transaction logs
+enable.log.retries.desc=When recovery logs are stored in an RDBMS table, this property allows SQL Operations that fail to be retried.
```

Comment:
looks good except I don't think "operations" needs to be capitalized 

When recovery logs are stored in an RDBMS table, this property allows SQL operations that fail to be retried.

Reply:
Thanks, I've updated the metatype properties in a new commit.